plugins/basic/python2.6libs/pythonrc.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+This file is loaded automatically by Houdini at startup.
+"""
+
+import inspect
+import os
+import sys
+
+def plugin_startup():
+
+    # use inspect to get the current file path since attempts to access
+    # __file__ result in a NameError.
+    current_file_path = os.path.abspath(
+        inspect.getsourcefile(lambda: 0)
```

Comment:
FYI, http://meta.stackexchange.com/questions/272956/a-new-code-license-the-mit-this-time-with-attribution-required?cb=1